### DISCUSSION TITLE
Is it possible to cluster MarkerViews?

### DISCUSSION THREAD

**Discussion by krakz999** [rnmapbox/maps#1278] [Q&A]
I'd like to have clustered markers that I can click on to bring up a callout. I've seen that there is a `MarkerView` component but it does not seem to be compatible with clustering.

I've checked the earthquakes demo to learn how the cluster works, but it only renders static points.

Is it possible to have interactive markers inside the clusters? Thanks

**Comment by mfazekas** [comment]
@krakz999 it's not supported ot of the box, you can try to use `supercluster` library to implement clustering with `MarkerView`
